components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -56,11 +56,15 @@ open class GleanInternalAPI {
      *
      * @param applicationContext [Context] to access application features, such
      * as shared preferences
+     * @param appBuild The build id of the application.
+     * @param appDisplayVersion The user visible version number of the application.
      * @param configuration A Glean [Configuration] object with global settings.
      * @raises Exception if called more than once
      */
     fun initialize(
         applicationContext: Context,
+        appBuild: String,
```

Comment:
@georgf , we're requiring the build id and display version to be passed to the `Glean.initialize` function. What's your take on this?

Reply:
I wonder what I'd pass in here as a developer though. I'd guess the `versionCode` of the app in most cases.

Reply:
@pocmo is it always like that for Android? Do you know how Focus or similar products work wrt this? If all that's used is `versionCode` and `versionNumber`, I think we can drop the parameters here and just always set them internally. That would be better, IMHO, so that no change is required by consumers :)

Reply:
The version code is what we report in the current telemetry library as "build id" (in the upload path). It's the same ID google play uses to identify a build. It's incremental and generated on taskcluster when building the application.

Reply:
Thanks Sebastian! I've changed the code so that we automatically get the version name and code without requiring input at init.